src/ServiceManagement/StorSimple/Commands.StorSimple/StorSimpleCmdletBase.cs

```diff
@@ -0,0 +1,335 @@
+﻿using System;
+using System.Linq;
+using System.Collections.Generic;
+using System.Threading;
+using Hyak.Common;
+using Microsoft.Azure;
+using Microsoft.WindowsAzure.Commands.StorSimple.Encryption;
+using System.Xml.Linq;
+using Microsoft.WindowsAzure.Management.StorSimple.Models;
+using Microsoft.WindowsAzure.Commands.Utilities.Common;
+using System.Net;
+using System.Management.Automation;
+using Microsoft.WindowsAzure;
+
+namespace Microsoft.WindowsAzure.Commands.StorSimple
+{
+    using Properties;
+    using Microsoft.WindowsAzure.Commands.StorSimple.Exceptions;
+    using Microsoft.WindowsAzure.Commands.StorSimple.Models;
+
+    public class StorSimpleCmdletBase : AzurePSCmdlet
+    {
+        private PSStorSimpleClient storSimpleClient;
+
+        internal PSStorSimpleClient StorSimpleClient
```

Comment:
We don't ass `PS` prefix to the convenience clients as they are supposed to be generic enough for any C# code not on PowerShell